tests/unit/test_main.py

```diff
@@ -1,18 +1,93 @@
 """Test the Main class"""
 from unittest import TestCase
 
-from kytos.lib.helpers import get_controller_mock
+from kytos.lib.helpers import get_controller_mock, get_test_client
+from napps.kytos.of_multi_table.main import Main
 
 
 class TestMain(TestCase):
     """Test the Main class"""
 
+    API_URL = "http://localhost:8181/api/kytos/of_multi_table"
```

Comment:
I haven't discussed with the team yet, I'll cover it today on weekly meeting but we should avoid introducing more test suites inheriting from `unittest.TestCase`. We use `pytest`, and `unittest.TestCase` doesn't have first class support for `async` test which will become more common with the upcoming changes that we have for this version. So, `unittest.TestCase` is pretty much a technical debt, so if you could not inherent from it, and then replace `def setUp(self)` with `def setup_method(self)` and get rid of all of the `self.assert.*` calls with `assert == ` that'd be great.